pages/docs/configuration/configuration-coupling-multi.md

```diff
@@ -11,28 +11,74 @@ preCICE allows us to combine multiple coupling schemes by only specifying them o
 
 ```xml
 <coupling-scheme:parallel-explicit>
-  <participants first="MySolver1" second="MySolver2"/>
+  <participants first="A" second="B"/>
   ...
 </coupling-scheme:parallel-explicit>
 <coupling-scheme:parallel-explicit>
-  <participants first="MySolver1" second="MySolver3"/>
+  <participants first="C" second="A"/>
   ...
 </coupling-scheme:parallel-explicit>
 ```
 
-For this example, all three participants are executed in parallel to one another, whereas `MySolver1` exchanges data with `MySolver2` and `MySolver3`, but not the latter two with each other. To also get an interaction between `MySolver2` and `MySolver3`, add a third coupling scheme.
+For this example, all three participants are executed in parallel to one another, whereas `A` exchanges data with `B` and `C`, but not the latter two with each other.
```

Comment:
```suggestion
For this example, all three participants are executed in parallel to one another, whereas `A` exchanges data with `B` and `C`, but `B` and `C` do not exchange data with each other.
```
I know it says the same, but I think it's better to make it even more clear. 

Maybe it will be nice to add a comment that "first" and "second" doesn't matter?